package.json

```
@@ -25,20 +25,35 @@
     "lint:fix": "npm run typecheck && eslint src --fix && prettier --w .",
     "prerelease": "npm run build",
     "prepublishOnly": "bun scripts/before-publish.ts",
-    "test": "npx braintrust eval src/tools-evaluations"
+    "test": "npx braintrust eval src/tools-evaluations",
```

`npx` is redundant here?
```suggestion
    "test": "braintrust eval src/tools-evaluations",
```

---

I was the one adding this here -  tbh, I think I added it due to some weird error and just kept it, but I can't remember what was it 🤔